tools/pyclient/dev/dev.py

```diff
@@ -0,0 +1,85 @@
+#///////////////////////////////////////////////////////////////////////////////
+# FILE: dev.py
+# AUTHOR: David Ruvolo
+# CREATED: 2023-05-22
+# MODIFIED: 2023-07-21
+# PURPOSE: development script for initial testing of the py-client
+# STATUS: ongoing
+# PACKAGES: pandas
+# COMMENTS: Designed to interact with the schema "pet store".
+#///////////////////////////////////////////////////////////////////////////////
+
+import pandas as pd
+
+from tools.pyclient.src.molgenis_emx2_pyclient import Client
+
+# connect and sign in
+db = Client('https://emx2.dev.molgenis.org/')
+db.sign_in('', '')
+
+# check status
+db.status()
+
+# get data
+data = db.get(schema='', table='') # run without specifying target
+data = db.get(schema = 'pet store', table='') # run without specifying table
+data = db.get(schema = 'pet store', table='Pet') # get Pets
+data
+
+data = db.get(schema = 'pet store', table='Pet', as_df=True) # get Pets
+
+#///////////////////////////////////////////////////////////////////////////////
+
+# ~ 1 ~
+# Check Import Methods
+
+# ~ 1a ~
+# Check import via the `data` parameters
+# Add new record to the pet store with new tags
+
+newTags = [
+    {'name': 'brown', 'parent': 'colors'},
+    {'name': 'canis', 'parent': 'species'},
+]
+
+newPets = [{
+    'name': 'Woofie',
+    'category': 'dog',
+    'status': 'available',
+    'weight': 6.8,
+    'tags': 'brown,canis'
+}]
+
+# import new data
+db.add(schema='pet store', table='Tag', data=newTags)
+db.add(schema='pet store', table='Pet', data=newPets)
+
+
+# retieve records
+data = db.get(schema='pet store', table='Pet')
+data
+
+# drop records
+tagsToRemove = [{'name': row['name']} for row in newTags if row['name'] == 'canis']
+db.delete(schema='pet store', table='Pet', data=newPets)
+db.delete(schema='pet store', table='Tag', data=tagsToRemove)
+
+#///////////////////////////////////////
+
+# ~ 1b ~
+# Check import via the `file` parameter
+
+# save datasets
+pd.DataFrame(newTags).to_csv('dev/demodata/Tag.csv', index=False)

```

Comment:
what does index=False mean?

Reply:
By default, pandas will write the row names (which are 0, 1, 2, 3, etc.). This option disables it to avoid issues where a table may have a column "index" and it is overwritten by the row names.